extensions/wikidata/src/org/openrefine/wikidata/qa/Constraint.java

```diff
@@ -0,0 +1,40 @@
+package org.openrefine.wikidata.qa;
+
+import org.wikidata.wdtk.datamodel.interfaces.Snak;
+import org.wikidata.wdtk.datamodel.interfaces.SnakGroup;
+import org.wikidata.wdtk.datamodel.interfaces.Statement;
+import org.wikidata.wdtk.datamodel.interfaces.Value;
+
+import java.util.List;
+import java.util.Set;
+
+public class Constraint {
```

Comment:
How about adding Javadocs to explain the role of this class?

Reply:
Yeah, sure!